tests/snapshots/tests.integration.test_integration_snapshots.test_tracer_trace_across_multiple_forks.snap

```diff
@@ -36,5 +38,6 @@
            "duration" 86000
            "meta" {"runtime-id" "859c494affcb4933a1e15b6c4d2561b8"}
            "metrics" {"_sampling_priority_v1" 1
+                      "_dd.top_level" 1
```

Comment:
This doesn't seem correct does it?

@Kyle-Verhoog what do you think the expected behavior is here?

Same with the previous "across fork" one as well.